ssl/quic/quic_impl.c

```diff
@@ -169,97 +167,222 @@ static int quic_raise_non_normal_error(QCTX *ctx,
                                 (reason),                       \
                                 (msg))
 
+#define QCTX_C              (1U << 0)
```

Comment:
I was immediately looking for the docs for these in terms of comments - but they are later on ... I think that later block comment should be moved above this or near it at least - so it is clearer to the user.

Reply:
Good point, thanks.